lib/fluent/plugin/formatter_csv.rb

```diff
@@ -35,6 +35,22 @@ class CsvFormatter < Formatter
       config_param :fields, :array, value_type: :string
       config_param :add_newline, :bool, default: true
 
+      def csv_cacheable?
+        owner ? true : false
```

Comment:
```suggestion
        not owner.nil?
```

Simpler?

Reply:
Hmm, I think using ternary operator is readable for me...

Reply:
one more suggestion: `!!owner`

Reply:
Thanks. Replaced with `!!owner`
https://github.com/fluent/fluentd/pull/4864/commits/eae43cae2f08bb94d4601599129ee416bb7038e3